NEWS.rst

```diff
@@ -1,3 +1,51 @@
+Version 0.12.0, 2018-11-16
+===============================
+
+New features
+------------
+
+* #23928: Updated auth to use Flask SAML SSO module.
+  Session is now shared between OS2MO and LoRa.
+* #22382: Manager hierarchy - the service returns all managers in a
+  hierarchical order
+* #24077: Access addresses and addresses from Dansk Adresseregister will
```

Comment:
“We now support access addresses in addition to regular addresses from Dansk Adresseregister, with combined autocompletion of the two.”